packaging/debian-sid/rules

```diff
@@ -17,6 +17,8 @@ export DH_GOLANG_EXCLUDES=tests
 export DH_GOLANG_GO_GENERATE=1
 
 export PATH:=${PATH}:${CURDIR}
+# GOCACHE is needed by go-1.13
+export GOCACHE:=/tmp/go-build
```

Comment:
do we need this for 14.04 too?

Reply:
IIRC this was only introduced in 1.12 or 1.13, so we probably do not need that on Trusty if building with go from the repositories.

Reply:
We also have this in packaging/ubuntu-14.04/rules

Reply:
ah right good point it's already there